src/main/scala/mesosphere/marathon/event/Events.scala

```diff
@@ -206,6 +207,16 @@ case class AppTerminatedEvent(
   eventType: String = "app_terminated_event",
   timestamp: String = Timestamp.now().toString) extends MarathonEvent
 
+case class UnhealthyTaskKillEvent(
```

Comment:
1) Cool. Please add to this PR (I do not see that change)
2) What I originally meant was even simpler: can you define that class under https://github.com/mesosphere/marathon/blob/master/src/main/scala/mesosphere/marathon/event/Events.scala#L118